test/FactorialServiceSpec.js

```diff
@@ -15,10 +15,18 @@ describe('FactorialService.js', function() {
     
     it('should null if an invalid param was given', function() {
         expect(FactorialService.calculate(-1)).toBe(null);
-        expect(FactorialService.calculate("1")).toBe(null);
```

Comment:
Edy, the input used is of type text, then in my opinion should accept instead a string equal to "5", for example. Since it can not accept string, declaring the type integer would not be more appropriate? thank you for attention. :)